src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -90,7 +90,7 @@ class SignInModal extends React.Component {
           'We’re sorry. We’re working to fix some problems with our My HealtheVet sign in process. If you’d like to sign in to VA.gov with your My HealtheVet username and password, please check back later.',
         )}
         {this.downtimeBanner(
-          [EXTERNAL_SERVICES.mvi],
+          [EXTERNAL_SERVICES.mvi, EXTERNAL_SERVICES.global],
```

Comment:
the signin modal provides specific content for each downtime type. I just reused the mvi copy because it's the most generic.